api/v1/api.go

```diff
@@ -207,7 +207,7 @@ func (s *apiV1) RegisterRoutes(e *echo.Echo) {
 	netw.GET("/addrs", s.handleNetAddrs)
 
 	miners := public.Group("/miners")
-	miners.GET("", s.handleAdminGetMiners)
+	miners.GET(":lookup_chain", s.handleAdminGetMiners)
```

Comment:
is this supposed to be under `/admin/miners`? My concern with putting this on /public is that someone can make unlimited calls to this endpoint which can affect the API's performance. We should instead cache the results and have an expiration of 1 hour.

Reply:
Good catch, yes I meant to put it on `/admin`, will fix!